src/components/scenes/Loans/ManageCollateralScene.tsx

```diff
@@ -43,22 +43,42 @@ import { FormScene } from '../FormScene'
 
 type ActionOpType = 'loan-borrow' | 'loan-deposit' | 'loan-repay' | 'loan-withdraw'
 
+// User input display strings
+const ACTIONOP_TYPE_STRING_MAP = {
+  'loan-borrow': {
+    headerText: s.strings.loan_borrow_more,
+    amountCard: s.strings.loan_fragment_loan,
+    srcDestCard: s.strings.loan_fund_destination
+  },
+  'loan-deposit': {
+    headerText: s.strings.loan_add_collateral,
+    amountCard: s.strings.loan_fragment_deposit,
+    srcDestCard: s.strings.loan_fund_source
+  },
+  'loan-repay': {
+    headerText: s.strings.loan_make_payment,
+    amountCard: s.strings.loan_fragment_repay,
+    srcDestCard: s.strings.loan_fund_source
+  },
+  'loan-withdraw': {
+    headerText: s.strings.loan_withdraw_collateral,
+    amountCard: s.strings.loan_fragment_withdraw,
+    srcDestCard: s.strings.loan_fund_destination
+  }
+}
```

Comment:
We can add `as const` here to make certain the type is strictly this constant value.

Reply:
Force pushed these changes